doc/manual/src/glossary.md

```diff
@@ -21,6 +21,24 @@
 
   [store derivation]: #gloss-store-derivation
 
+- [store derivation output reference]{#gloss-store-derivation-output-reference}
+
+  The string resulting from concatenating three things:
+
+  - The path to a [store derivation](#gloss-store-derivation)
+  - The character `^`
+  - Either:
+    - one of the store derivation's [output names](#gloss-output-name) or
+    - the character `*`
+
```

Comment:
```suggestion

  A plain store derivation path refers to the first output declared for that derivation.

```

Reply:
Hrm, @fricklerhandwerk did you mean to delete those lines in your suggestion, or did you intend to only add the new line?